scripts/log-capture

```diff
@@ -29,20 +29,20 @@ _to_log dmesg
 _to_log dmidecode
 _to_log lspci -vvnn
 
-for DEV_NAME in $(list-harddrives | cut -d " " -f 1)
```

Comment:
@M4rtinK if this not makes usage of `list-harddrives` deprecated? After checking code `log-capture` looks to be only place where it is being called.

Reply:
Oh, right, `list-harddrives` - indeed, it might look like that on the first glance, but actually the main use-case is scriptlets in kickstart: 

https://anaconda-installer.readthedocs.io/en/latest/list-harddrives.html

Eq. its used for user kickstarts that need to handle some custom storage needs. 

The name is distinct enough, so I was even able to find a couple random kickstart by GitHub code search:

https://github.com/mrsipan/ksfiles/blob/6f5b02f3c1fc8062bf482aa13bfaff5e10692833/centos.ks#L28

https://github.com/Heretic-oVirt/kickstart/blob/d05a8f0c64769fc77aae9ec3d79b0c546bbd3c06/heretic-ngn.ks#L549

https://github.com/NethServer/nethserver-proxmox-template/blob/1bdfc0a711175656a5d89097a1b5f85ca4c857ba/config/ks.cfg#L13